borg/testsuite/logger.py

```diff
@@ -0,0 +1,43 @@
+import logging
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
there were talks in the issue queue of backporting to 2.x, no? just taking precautions...